src/main/java/connectingstar/tars/user/domain/User.java

```diff
@@ -0,0 +1,46 @@
+package connectingstar.tars.user.domain;
+
+import jakarta.persistence.*;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    private String nickname; //닉네임
+
+    private String gender; //남,여,선택안함
```

Comment:
```suggestion
    private String gender; //남,여,선택안함
```
gender의 경우의 수는 남, 여, 선택안함으로 총 3가지 경우의 수만 존재하기 때문에 해당 컬럼 역시 Enum으로 정의하여 사용하면 좋을 것 같습니다.

간단하게 M(Male), F(Female), N(None)으로 구성하여 DB 컬럼 타입을 varchar이 아닌 char 타입으로 정의하면 되겠네용

Reply:
이것도 그럼 @Enumerated(EnumType.STRING) 으로 하면 되겠죠?